### PR TITLE
do not fail on unreadable directories

### PR DESCRIPTION
**Pull request by anarcat**
this will still warn us, but at least not completely crash

this is stacked on top of #26

### CODE REVIEW

**Comment by anarcat** [comment]
so it seems this doesn't actually work, but i can't quite figure out why... it seems that glob.sync() still just doesn't return anything, even with `strict: false`. grml.

**Comment by anarcat** [comment]
nevermind that, it's the bare pattern that's screwed up... lots of fun, that glob library... :/

**Comment by anarcat** [comment]
alright, this glob pattern actually works, and the error suppression too.

sorry about the noise.

**Comment by cjb** [comment]
Cool.  Is the added `console.log` intentional?

**Comment by anarcat** [comment]
it's not. a leftover of debugging :( want me to remove that? i am thinking it's pretty useful to have a one-liner list of repos, it certainly helped me to understand wtf was going on here :)

**Comment by cjb** [comment]
I don't think we need both the one-line list and a separate console.log for each one, so I'd remove one or the other, thanks.

**Comment by anarcat** [comment]
alright, done!

**Comment by cjb** [comment]
Merged, thanks Antoine!

@splinterofchaos, if you still have changes, maybe base them on current master now and ask to merge?  Thanks!

**Comment by anarcat** [comment]
thanks, @cjb

File: gittorrentd [hunk 1]
```diff
@@ -76,7 +76,7 @@ function bpad (n, buf) {
 
 dht.on('ready', function () {
   // Spider all */.git dirs and announce all refs.
-  var repos = glob.sync('*/.git/git-daemon-export-ok')
```
Comment (wolk935) [inline gittorrentd]: console.log('in repo ' + repo)
    repo = repo.replace(/git-daemon-export-ok$/, '')

**Pull request merged**